src/Effect/Foldable.agda

```diff
@@ -0,0 +1,77 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Foldable functors
+------------------------------------------------------------------------
+
+-- Note that currently the Foldable laws are not included here.
+
+{-# OPTIONS --cubical-compatible --safe #-}
+
+module Effect.Foldable where
+
+open import Algebra.Bundles.Raw using (RawMonoid)
+open import Algebra.Bundles using (Monoid)
+import Algebra.Construct.Flip.Op as Op
+
+open import Data.List.Base as List using (List; [_]; _++_)
+
+open import Effect.Functor as Fun using (RawFunctor)
+
+open import Function.Base using (id; flip)
+open import Function.Endomorphism.Propositional using (∘-id-monoid)
```

Comment:
Something went wrong here... should be `Endo` since #2342 but my commit history on this branch had got out of sync somehow?

Reply:
Hopefully fixed now!